### PR TITLE
Test with GitHub Actions. Support Python 3.9

### PR DESCRIPTION
**Pull request by mauritsvanrees**
Let's see if this already kicks in when creating this pull request.

### CODE REVIEW

**Comment by mauritsvanrees** [comment]
I force pushed a lot of times to get this right, but now I think it is ready. I am happy to merge it, as the tests pass. But a second look would be good.

GitHub Actions is quite fast, but of course for `i18ndude` we only need to install a few dependencies, not the whole of Plone, so that helps a lot.

We already had a Tox config for local testing, which makes it easier too. I think that is the way to go for most packages that need GitHub Actions testing.

Also see https://github.com/zopefoundation/meta/blob/master/config/default/tests.yml.j2 for how Zope packages are starting to use this.

File: .github/workflows/main.yml [hunk 1]
```diff
@@ -0,0 +1,36 @@
+name: Tests
+on:
+  # Test all pull requests, and any push to master.
+  pull_request:
+  push:
+    branches:
+      - master
+  schedule:
+    # run once a month on master branch
+    - cron: '0 12 10 * *'
```
Comment (ale-rt) [inline .github/workflows/main.yml:10]: I always see this same timestamp in every workflow... :)

File: tox-check-i18ndude-command.sh [hunk 2]
```diff
@@ -0,0 +1,32 @@
+#!/bin/sh
+# Shell script to test the i18ndude command in tox.
+# In tox commands you cannot use shell constructs like double pipes.
```
Comment (ale-rt) [inline tox-check-i18ndude-command.sh:3]: Should we move this in the test folder?

File: README.rst [hunk 3]
```diff
@@ -9,9 +9,6 @@ Introduction
 
 i18ndude performs various tasks related to ZPT's, Python Scripts and i18n.
 
-.. image:: https://secure.travis-ci.org/collective/i18ndude.png?branch=master
-    :alt: Travis CI badge
-    :target: http://travis-ci.org/collective/i18ndude
```
Comment (ale-rt) [inline README.rst:14]: We could replace the travis badge with the GA one: 
```
.. image:: https://github.com/collective/i18ndude/workflows/Tests/badge.svg
    :alt: GitHub Actions badge
    :target: https://github.com/collective/i18ndude/actions
```

**Review by ale-rt — APPROVED** [review]
LGTM, just minor remarks.

File: .github/workflows/main.yml (hunk 1 above)
Comment (mauritsvanrees) [reply .github/workflows/main.yml:10]: I saw '0 12 * * 0'  for running once a week on Sunday. I do it once a month, on day 10. So it is different. :-)
But yes, might be better to randomise this a bit. I will do that.

File: README.rst (hunk 3 above)
Comment (mauritsvanrees) [reply README.rst:14]: Ah, nice. Done.

File: tox-check-i18ndude-command.sh (hunk 2 above)
Comment (mauritsvanrees) [reply tox-check-i18ndude-command.sh:3]: That could make things easier. Let me try.

**Comment by mauritsvanrees** [comment]
@ale-rt I have updated based on your comments.

**Review by ale-rt — APPROVED** [review]
Lovely!

**Pull request merged**